### ISSUE TITLE
Ask not to remove the code: -webkit-box-orient

### ISSUE THREAD

**Issue by mozibrand** [postcss/autoprefixer#1141]
By default, Autoprefixer also removes outdated prefixes.This is not out of date, consider not removing it

**Comment by ai** [comment]
Yeap. I know about `lin-claimp` case. I personally don’t like webkite-only hacks. So I do not want to spend my time to allow people to do them (but it is still possible by `/* autoprefixer: ignore next */`).

But I will accept PR if somebody will fix it.

**Comment by ai** [comment]
Fixed by 69ed8a0 and released in 9.6.1.

**Comment by johnnybenson** [comment]
```css
        display: -webkit-box; /* stylelint-disable-line value-no-vendor-prefix */
        -webkit-line-clamp: 2;
        -webkit-box-orient: vertical; /* stylelint-disable-line property-no-vendor-prefix */
        overflow: hidden;
```

if anyone is looking
